Documentation/PackageDescription.md

```diff
@@ -306,6 +306,16 @@ static func package(url: String, _ requirement: Package.Dependency.Requirement)
 ///     - branch: A dependency requirement. See static methods on `Package.Dependency.Requirement` for available options.
 static func package(name: String? = nil, url: String, branch: String) -> Package.Dependency
 
+/// Adds a remote package dependency given a branch requirement.
+///
+///    .package(url: "https://example.com/example-package.git", revision: "aa681bd6c61e22df0fd808044a886fc4a7ed3a65"),
+///
+/// - Parameters:
+///     - name: The name of the package, or nil to deduce it from the URL.
+///     - url: The valid Git URL of the package.
+///     - revision: A dependency requirement. See static methods on `Package.Dependency.Requirement` for available options.
```

Comment:
This doc comment probably needs to be updated.  I think the rev can be a hash or a tag.